coremltools/converters/mil/test_flexible_shape_inputs.py

```diff
@@ -84,6 +88,25 @@ def test_multiarray_input_rangedim(self, convert_to):
         assert spec.description.input[0].type.multiArrayType.shapeRange.sizeRanges[2].upperBound == 100
         _assert_torch_coreml_output_shapes(model, spec, traced_model, example_input)
 
+    @pytest.mark.parametrize("convert_to", ['neuralnetwork', 'mlprogram'])
+    def test_multiarray_input_rangedim_flatten(self, convert_to):
+        if convert_to == "mlprogram" and ct.utils._macos_version() < (12, 0):
```

Comment:
Why are we skipping this test for mlprogram?

Reply:
I just copied from other test and skipping is not needed for this test. thanks.